djangosaml2idp/views.py

```diff
@@ -414,18 +435,20 @@ def get(self, request: HttpRequest, *args, **kwargs):
         return self.render_response(request, html_response, None)
 
 
+@method_decorator(never_cache, name="dispatch")
+class MetadataView(IdPHandlerViewMixin, View):
```

Comment:
Moved as a class based view for inheriting the dynamic configuration loading method